### ISSUE TITLE
Also no minecraft mod? 

### ISSUE THREAD

**Issue by jryebread** [Poppro/MCNE#2]

**Comment by frisby-ninja** [comment]
Yeah, I'm thinking of rewriting the mod (maybe a Baritone implementation for smoother visualizations?). Will definitely post the source code if I do manage it, but some help or info would be very appreciated!
